app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/supportingevidence/holding_page.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.AppConfig
+@import play.api.i18n.Messages
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components._
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html._
+
+@this(layout: Layout, submit_button: submit_button)
```

Comment:
rename `submit_button` to `submitButton` - `submit_button` is the component, `submitButton` is the reference to that component.